staging/src/k8s.io/client-go/transport/cache.go

```diff
@@ -51,7 +52,7 @@ func (t tlsCacheKey) String() string {
 	if len(t.keyData) > 0 {
 		keyText = "<redacted>"
 	}
-	return fmt.Sprintf("insecure:%v, caData:%#v, certData:%#v, keyData:%s, serverName:%s", t.insecure, t.caData, t.certData, keyText, t.serverName)
+	return fmt.Sprintf("insecure:%v, caData:%#v, certData:%#v, keyData:%s, getCert: %s, serverName:%s", t.insecure, t.caData, t.certData, keyText, t.getCert, t.serverName)
```

Comment:
Hmmm, how does this work with getCert?

Reply:
It uses the function address to uniquely identify the cached http.Transport.
It's not really calling GetCert.